lib/workers/repository/finalise/prune.ts

```diff
@@ -64,6 +76,14 @@ async function cleanUpBranches(
         logger.info(
           `PRUNING-DISABLED: Would deleting orphan branch ${branchName}`
         );
+      } else if (platform.supportsPrReopen) {
+        if (!closedRightNow) {
+          const lastCommitTime = await getBranchLastCommitTime(branchName);
+          const minutesFromLastCommit = moment().diff(lastCommitTime, 'm');
+          if (minutesFromLastCommit >= 60) {
```

Comment:
@viceice @rarkins 
What do you think about reasonable interval?

Reply:
This means 1 hour between PR closed and the branch being deleted? Seems reasonable.